mmv1/third_party/terraform/services/dns/resource_dns_record_set.go

```diff
@@ -16,6 +16,16 @@ import (
 	"google.golang.org/api/dns/v1"
 )
 
+func lbTypeNoneDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
+	// Check if the key matches the relevant attribute
+	if k != "routing_policy.0.primary_backup.0.primary.0.internal_load_balancers.0.load_balancer_type" {
```

Comment:
So, `internal_load_balancers` is a list, which means this diff suppress func will only function on the first element of that list. Can you add a test that specifies 2 `internal_load_balancers` that would need to use this diff suppress func?